From 0c9b6f8109dec616b5c03c56391cf6e2795da726 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Tue, 5 Nov 2013 15:15:17 +0000
Subject: [PATCH] builder/website: Add validate.sh test script to EXTRA_DIST.

This fixes commit 154bfb7dd46e7998eed1d40ead95923590763739.

(cherry picked from commit 92206e8d09455b3ec63d44e4b55d5eef415681af)
---
 builder/website/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/builder/website/Makefile.am b/builder/website/Makefile.am
index b45d98a..680fd54 100644
--- a/builder/website/Makefile.am
+++ b/builder/website/Makefile.am
@@ -21,6 +21,7 @@ EXTRA_DIST = \
 	.gitignore \
 	compress.sh \
 	test-guest.sh \
+	validate.sh \
 	README \
 	index \
 	index.asc \
-- 
1.8.3.1

